web/public/config/config.json

```diff
@@ -25,7 +25,7 @@
     "successText": "To try out your new credential:",
     "links": [
       {
-        "url": "https://csb-audio.pathfinder.gov.bc.ca",
+        "url": "https://csb-audio.apps.silver.devops.gov.bc.ca",
```

Comment:
Please make this `https://csb-audio-demo-test.apps.silver.devops.gov.bc.ca`

Reply:
change completed